daemon/daemon_test.go

```diff
@@ -188,7 +188,7 @@ func TestContainerInitDNS(t *testing.T) {
 	hostConfig := `{"Binds":[],"ContainerIDFile":"","Memory":0,"MemorySwap":0,"CpuShares":0,"CpusetCpus":"",
 "Privileged":false,"PortBindings":{},"Links":null,"PublishAllPorts":false,"Dns":null,"DnsOptions":null,"DnsSearch":null,"ExtraHosts":null,"VolumesFrom":null,
 "Devices":[],"NetworkMode":"bridge","IpcMode":"","PidMode":"","CapAdd":null,"CapDrop":null,"RestartPolicy":{"Name":"no","MaximumRetryCount":0},
-"SecurityOpt":null,"ReadonlyRootfs":false,"Ulimits":null,"LogConfig":{"Type":"","Config":null},"CgroupParent":""}`
+"SecurityOpt":null,"ReadonlyRootfs":false,"Ulimits":null,"LogConfig":{"Type":"","Config":null},"CgroupParent":"",Capabilities:["CAP_KILL"]}`
```

Comment:
The fact that you had to change this test suggests that you changed the default config. You can't make that kind of change without it being backwards compatible, and you will need a full set of tests for this change. It is unfortunately difficult making this sort of change.

Reply:
Before I respond in full, let me make sure I understand your concern. I think what you are saying is: Changing the required default capabilities for everyone is a not something we want to do. 

If that is your concern then I do not think that is an issue, if you look at https://github.com/moby/moby/pull/39297/files#diff-72dc1842ed7b5a92b4e98f84456a2c0cR154
You can see that we are only setting the capabilities if they are configured in the daemon config. If they are not set we continue with the current default. (I reused the existing test but I can make a copy that is specific to my case if you would like.)

Cheers.

Reply:
definitely do not change an existing test.

Reply:
@justincormack Looks like the tests are working other than the Windows RS1 tests. I moved my test case into its own test and left the DNS test as it is.